### PR TITLE
Update Hermes to v1.2.6

### PR DESCRIPTION
**Pull request by JoshOldenburg**

### CODE REVIEW

File: Casks/hermes.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask :v1 => 'hermes' do
-  version '1.2.5'
-  sha256 '786e20040bdd6e943b646f74d7cc9a460b37444272c4a7af8d1f5ef77a7d2f2f'
+  version '1.2.6'
+  sha256 'aeecd452dc9b920abd467a6e8223db9e6c84771a'
```
Comment (vitorgalvao) [inline Casks/hermes.rb]: This is too short. How are you calculating it?

Comment (JoshTheGeek) [inline Casks/hermes.rb]: I used `shasum` with the default, which is SHA1. `shasum -a 256 <file>` gives the SHA256 sum. This is fixed in JoshTheGeek@495018f.

**Comment by fanquake** [comment]
Can you please squash your commits

**Comment by ndr-qef** [comment]
Thanks. Merged as e5aba21.

**Pull request closed without merging**